provision/openshift/Taskfile.yaml

```diff
@@ -91,6 +91,60 @@ tasks:
       - .task/var-KC_STORAGE
       - .task/var-KC_DATABASE
 
+  grafana-sa:
+    # this setup was inspired by https://zhimin-wen.medium.com/custom-grafana-dashboard-for-user-workload-in-openshift-6dc2d4cad274
+    deps:
+      - common:split
+      - common:env
+      - openshift-env
+    cmds:
+      - kubectl create namespace monitoring || true
+      # this creates the service account outside the Helm chart to be able to create the token which is later an input of the Helm change
+      - oc create sa grafana -n monitoring || exit 0
+      - oc adm policy add-cluster-role-to-user cluster-monitoring-view -z grafana -n monitoring
+      - bash -c 'test -e .task/grafana-sa-token || (oc create token grafana -n monitoring > .task/grafana-sa-token)'
+      - kubectl get project/monitoring -o jsonpath='{.metadata.annotations.openshift\.io/sa\.scc\.uid-range}' |  cut -f1 -d"/" > .task/monitoring-uids
+    sources:
+      - .task/subtask-{{.TASK}}.yaml
+    status:
+      - test -f .task/grafana-sa-token
```

Comment:
I did some research, and I think those tokens don't expire. Let's keep an eye on this. 

This is the link I found: https://cookbook.openshift.org/accessing-an-openshift-cluster/how-can-i-create-a-service-account-for-scripted-access.html

Maybe instead of creating a new service account token on the CLI, I should pick the one created in OpenShift instead. If we see problems there, I'll refine it.

Reply:
I agree with follow-up.